### DISCUSSION TITLE
Loading German model takes for ever

### DISCUSSION THREAD

**Discussion by lovodkin93** [explosion/spaCy#6753] [Help: Coding & Implementations]
Hello,
I am trying to load the German model, and when I run it from my pycharm, it takes it 20-30 seconds, but it loads without any problem. But, when I try to run it from the clusters, it just won't load (it has been trying to load now for over 10 minutes and hasn't finished).
Has anyone encountered such a problem?

**Comment by svlandeg** [comment]
Hi! I've converted your issue to a discussion on the new discussions board, as this feels like a topic where it would be beneficial for the community to chime in.

Can you give a bit more information what your architecture is where it's not working (or taking very long)? What are "the clusters" and how are you executing your script(s)?
